exercises/secret-handshake/src/example/kotlin/Signal.kt

```diff
@@ -0,0 +1,5 @@
+enum class Signal {
```

Comment:
I prefer to avoid the duplication, but not really tied to either. This PR looks good either way.